lib/presentation/widgets/icon_text_button_widget.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:flutter/material.dart';
+
+class IconTextButton extends StatefulWidget {
+  final String text_left_side;
```

Comment:
lowerCamelCase here

Reply:
and everywhere in the project when you are declaring variables, pay more attention on that

Reply:
corrected successfully